### ISSUE TITLE
Failed to build iOS app - error: 'keyboardWillShowNotification' has been renamed to     'NSNotification.Name.UIKeyboardWillShow'

### ISSUE THREAD

**Issue by ebot64** [GiancarloCode/form_bloc#70]
After adding flutter_form_bloc: ^0.12.3 to my project's pubspec.yaml file, the project fails to build and run on iOS simulator, but runs well on android. Does flutter_form_bloc depend on a swift plugin that is probably the cause of this. The Error message is shown below. 

```
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/ebot64/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/keyboard_utils-1.2.0/ios/Classes/SwiftKeyboardUtilsPlugin.swift:49:66: error: 'keyboardWillShowNotification' has been renamed to
    'NSNotification.Name.UIKeyboardWillShow'
                                                   name: UIResponder.keyboardWillShowNotification,
                                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                     NSNotification.Name.UIKeyboardWillShow
    UIKit.UIResponder:3:22: note: 'keyboardWillShowNotification' was introduced in Swift 4.2
        public class let keyboardWillShowNotification: NSNotification.Name
                         ^
    /Users/ebot64/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/keyboard_utils-1.2.0/ios/Classes/SwiftKeyboardUtilsPlugin.swift:54:66: error: 'keyboardWillHideNotification' has been renamed to
    'NSNotification.Name.UIKeyboardWillHide'
                                                   name: UIResponder.keyboardWillHideNotification,
                                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                     NSNotification.Name.UIKeyboardWillHide
    UIKit.UIResponder:7:22: note: 'keyboardWillHideNotification' was introduced in Swift 4.2
        public class let keyboardWillHideNotification: NSNotification.Name
                         ^
    /Users/ebot64/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/keyboard_utils-1.2.0/ios/Classes/SwiftKeyboardUtilsPlugin.swift:59:66: error: 'keyboardDidShowNotification' has been renamed to
    'NSNotification.Name.UIKeyboardDidShow'
                                                   name: UIResponder.keyboardDidShowNotification,
                                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                     NSNotification.Name.UIKeyboardDidShow
    UIKit.UIResponder:5:22: note: 'keyboardDidShowNotification' was introduced in Swift 4.2
        public class let keyboardDidShowNotification: NSNotification.Name
                         ^
    /Users/ebot64/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/keyboard_utils-1.2.0/ios/Classes/SwiftKeyboardUtilsPlugin.swift:73:80: error: 'keyboardFrameEndUserInfoKey' has been renamed to
    'UIKeyboardFrameEndUserInfoKey'
                if let keyboardFrame: NSValue = notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue {
                                                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                                   UIKeyboardFrameEndUserInfoKey
    UIKit.UIResponder:15:22: note: 'keyboardFrameEndUserInfoKey' was introduced in Swift 4.2
        public class let keyboardFrameEndUserInfoKey: String
```

flutter doctor -v[✓] Flutter (Channel master, v1.18.0-dev.5.0, on Mac OS X 10.15 19A602, locale en-NG)
    • Flutter version 1.18.0-dev.5.0 at /users/user/documents/flutter
    • Framework revision 7f56b53de4 (4 days ago), 2020-04-12 12:00:01 -0400
    • Engine revision beb8a7ec48
    • Dart version 2.8.0 (build 2.8.0-dev.20.0 89b0f67261)

[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/user/Library/Android/sdk
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.1, Build version 11A1027
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 38.1.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✓] VS Code (version 1.44.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.9.1

[✓] Connected device (2 available)

• No issues found!

**Comment by GiancarloCode** [comment]
This is the same https://github.com/GiancarloCode/form_bloc/issues/65#issuecomment-613090642

I will change the plugin for detect keyboard, currently I don't have a Mac :(.

Could you please use this plugin and tell me if it detects the visibility of the keyboard in iOS,
If it works, let me know and I'll update in a couple of hours :)

https://pub.dev/packages/flutter_keyboard_visibility#-readme-tab-

**Comment by ebot64** [comment]
So, it happens that my project uses SWIFT_VERSION 4.0.  I created a new flutter project that defaults to SWIFT_VERSION 5.0, added the form_bloc plugin and the project ran correctly without this issue. 
Therefore, I added this line -> `config.build_settings['SWIFT_VERSION'] = '5.0'` to my project's `ios/podfile` and the build error is gone. Project builds and runs fine now.

Below is the portion of my ios/podfile with the SWIFT_VERSION included

```
# Prevent Cocoapods from embedding a second Flutter framework and causing an error with the new Xcode build system.
install! 'cocoapods', :disable_input_output_paths => true

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
      config.build_settings['SWIFT_VERSION'] = '5.0'
    end
  end
end

```

**Comment by GiancarloCode** [comment]
Thank you, can you please use the git version, I changed the keyboard package but i want to know if it work on iOS
```yaml
dependencies:
  flutter_form_bloc:
    git:
      url: git://github.com/GiancarloCode/form_bloc.git
      path: packages/flutter_form_bloc
```

**Comment by ebot64** [comment]
So, to be sure this works as expected, I commented out the SWIFT VERSION configuration that solved the problem --> **`#config.build_settings['SWIFT_VERSION'] = '5.0'`** and then ran in iOS to be sure the error is there still; and yes the keyboard error did return.

Then I used the git version as you instructed, happy to say the error is **GONE**.

> Thank you, can you please use the git version, I changed the keyboard package but i want to know if it work on iOS
> 
> ```yaml
> dependencies:
>   flutter_form_bloc:
>     git:
>       url: git://github.com/GiancarloCode/form_bloc.git
>       path: packages/flutter_form_bloc
> ```

**Comment by GiancarloCode** [comment]
Great, in a couple of hours I will publish the new version in pub :)

**Comment by GiancarloCode** [comment]
Solved in `flutter_form_bloc: 0.13.0`.
